release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1199,7 +1199,19 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Bare Metal Hardware Provisioning*
 
+* Because newer ipmitool packages default to using cipher suite 17, older hardware that does not support cipher suite 17 fails during deployment. When cipher suite 17 is not supported by the hardware, Ironic now uses cipher suite 3 so that deployments on older hardware using ipmitool should succeed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1897415[*BZ#1897415*])
 
+* Previously, after an adoption failure of externally provisioned hosts, no retry was attempted. In some cases, adoption occured before the image cache was populated, which resulted in permanent adoption failure. This caused the control plane bare metal hosts to report `adoption failed.` With this update, adoption of externally provisioned hosts automatically retries after adoption failure until control plane hosts are correctly adopted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905577[*BZ#1905577*])
+
+* Previously, the Custom Resource (CR) required the Baseboard Management Controller (BMC) details. However, with assisted installers this information was not provided. This update allows the CR to bypass the BMC details when the operator is not creating the nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913112[*BZ#1913112*])
```

Comment:
Do you know what CR is affected here. It looks odd to not have a type. (I see that the BZ and PR are lacking such detail). Are _details_ the BMC credentials? I see that in the docs a bit.